contracts/WildcardSteward_v2.sol

```diff
@@ -450,13 +454,23 @@ contract WildcardSteward_v2 is Initializable {
         emit RemainingDepositUpdate(patron, deposit[patron]);
     }
 
-    function buy(uint256 tokenId, uint256 _newPrice, uint256 _deposit)
+    function buy(
+        uint256 tokenId,
+        uint256 _newPrice,
+        uint256 _deposit,
+        uint256 wildcardsPercentage
+    )
         public
         payable
         collectPatronage(tokenId)
         collectPatronageAddress(msg.sender)
     {
         require(_newPrice > 0, "Price is zero");
+        require(
+            wildcardsPercentage >= 500 && wildcardsPercentage <= 10000,
```

Comment:
We need to list the artist cut as a variable (instead of it being hard coded as is, and remove this amount from the full 10000 percentage amount.